src/test/java/org/jenkinsci/plugins/workflow/multibranch/JobPropertyStepTest.java

```diff
@@ -211,4 +230,135 @@
+        p.setDefinition(new CpsFlowDefinition(
+                (HAVE_SYMBOL ?
+                        "properties([pipelineTriggers([\n"
+                                + "  cron('@daily'), [$class: 'MockTrigger']])])\n" :
+                        "properties([pipelineTriggers(\n"
+                                + "  triggers: [[$class: 'TimerTrigger', spec: '@daily'],\n"
+                                + "    [$class: 'MockTrigger']])])\n"
+                ) + "echo 'foo'"));
+
+        WorkflowRun b = r.assertBuildStatusSuccess(p.scheduleBuild2(0));
+
+        // Verify that we're seeing warnings due to running 'properties' in a non-multibranch job.
+        r.assertLogContains("WARNING: The 'properties' step will remove all 'JobProperty's currently configured in this job, either from the UI or from an earlier 'properties' step.", b);
+        // Verify that we're not seeing warnings for any properties being removed, since there are no pre-existing ones.
+        r.assertLogNotContains("WARNING: Removing existing job property", b);
+
+        assertEquals(2, p.getTriggers().size());
+
+        PipelineTriggersJobProperty triggerProp = p.getTriggersJobProperty();
+
+        TimerTrigger timerTrigger = getTriggerFromList(TimerTrigger.class, triggerProp.getTriggers());
+
+        assertNotNull(timerTrigger);
+
+        assertEquals("@daily", timerTrigger.getSpec());
+
+        MockTrigger mockTrigger = getTriggerFromList(MockTrigger.class, triggerProp.getTriggers());
+
+        assertNotNull(mockTrigger);
+
+        assertTrue(mockTrigger.isStarted);
+
+        assertEquals("[null, false]", MockTrigger.startsAndStops.toString());
+
+        // Now run a properties step with a different property and verify that we still have a
+        // PipelineTriggersJobProperty, but with no triggers in it.
+        p.setDefinition(new CpsFlowDefinition("properties([disableConcurrentBuilds()])\n"
+                + "echo 'foo'"));
+
+        WorkflowRun b2 = r.assertBuildStatusSuccess(p.scheduleBuild2(0));
+
+        // Verify that we're seeing warnings due to running 'properties' in a non-multibranch job.
+        r.assertLogContains("WARNING: The 'properties' step will remove all 'JobProperty's currently configured in this job, either from the UI or from an earlier 'properties' step.", b2);
+        // Verify that we *are* seeing warnings for removing the triggers property.
+        r.assertLogContains("WARNING: Removing existing job property 'Build triggers'", b2);
+
+        assertNotNull(p.getTriggersJobProperty());
+
+        assertTrue(p.getTriggers().isEmpty());
+
+        assertEquals("[null, false, null]", MockTrigger.startsAndStops.toString());
+    }
+
+    @Issue("JENKINS-37005")
+    @Test 
+    public void noPropertiesWarnings() throws Exception {
+        sampleRepo.init();
+        sampleRepo.write("Jenkinsfile", "echo \"branch=${env.BRANCH_NAME}\"\n"
+                + "properties([[$class: 'DisableConcurrentBuildsJobProperty']])");
```

Comment:
:ant: use `disableConcurrentBuilds` (as I think I mentioned before but you maybe rebased my comments out of existence)